projects/storefrontlib/src/translations/en/meta-resolver.en.ts

```diff
@@ -0,0 +1,25 @@
+export const metaResolver = {
+  categoryPage: {
+    heading: '{{count}} result for {{query}}',
+    heading_plural: '{{count}} results for {{query}}',
+  },
+  contentPage: {
+    heading: '{{content}}',
+  },
+  checkoutPage: {
+    heading: 'Checkout {{count}} item',
+    heading_plural: 'Checkout {{count}} items',
+  },
+  cartPage: {
+    heading: '{{title}} ({{code}})',
+  },
+  searchPage: {
+    heading: '{{total}} result for {{query}}',
```

Comment:
the variable must have name `count` to trigger the pluralization logic

Reply:
done